.github/workflows/build-package-docs.yaml

```diff
@@ -122,6 +122,27 @@ jobs:
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
 
+  notify-build-failures:
+    if: failure()
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    env:
+      ROOM_URL: https://ansible-accounts.ems.host/_matrix/client/v3/rooms/!HJtetIFWYEIDBOXxFE:libera.chat/send/m.room.message
```

Comment:
not sure about libera.chat here? But if it's sending bot messages to IRC, I'd say drop it for now. Nearly all the docs maintainers who could take action are not watching IRC anymore anyway.

Reply:
@samccann This is the internal room ID for Ansible Docs. I'm not sure why it's `libera.chat`. That's probably a question for Greg. In any event you need this piece of info for the API call.

https://spec.matrix.org/v1.4/client-server-api/#mroommessage

The room ID is returned with this API call:

```
curl -v https://ansible-accounts.ems.host/_matrix/client/v3/directory/room/%23docs:ansible.com

{"room_id":"!HJtetIFWYEIDBOXxFE:libera.chat","servers":["ansible.im","jb3.dev","matrix.org","libera.chat","ansible.com","fedora.im","doczkal.cloud","one.ems.host","gitter.im","tchncs.de","hu-berlin.de","kootstra.frl","matrix.thoummavong.de"]
```

You can also get the internal room ID from the settings (most other rooms have `libera.chat` in them):

![image](https://github.com/ansible/ansible-documentation/assets/32749632/83fe3845-0fe7-45b4-ae00-332b8351debb)